### ISSUE TITLE
🐛  K8s deployment: Process 101 waits for AccessExclusiveLock on relation 16701 of database 16385; blocked by process 112

### ISSUE THREAD

**Issue by harshithmullapudi** [airbytehq/airbyte#8276]
## Steps to Reproduce
1. Launch a new k8s airbyte instance
2. Import config using API 

To resolve it
1. kubectl delete all --all
2. kubectl delete -k kube/overlays/stable
3. deploy it again

**Comment by jrhizor** [comment]
@harshithmullapudi this means that all configs imports are not working on Kubernetes right now?

**Comment by jrhizor** [comment]
Is this a duplicate of https://github.com/airbytehq/airbyte/issues/8231?

**Comment by harshithmullapudi** [comment]
@jrhizor but it works when I try again. Is this expected that trying multiple times will make it work ?

**Comment by lmossman** [comment]
@harshithmullapudi I'm having trouble reproducing this issue. Would you be able to attach your configuration archive file to this issue (as long as it does not contain any sensitive secret information)?

Also, which pod were you seeing the following log from?
```
Process 101 waits for AccessExclusiveLock on relation 16701 of database 16385; blocked by process 112
```

**Comment by cgardens** [comment]
@harshithmullapudi we are going to close it. let us know if you see it again and can reproduce.
